diff --git a/node_modules/kaplay/src/kaplay.ts b/node_modules/kaplay/src/kaplay.ts
index 19ab4cf..b8f2692 100644
--- a/node_modules/kaplay/src/kaplay.ts
+++ b/node_modules/kaplay/src/kaplay.ts
@@ -266,8 +266,8 @@ import {
     toWorld,
 } from "./game";
 
-import boomSpriteSrc from "./kassets/boom.png";
-import kaSpriteSrc from "./kassets/ka.png";
+// import boomSpriteSrc from "./kassets/boom.png";
+// import kaSpriteSrc from "./kassets/ka.png";
 
 export let k: KAPLAYCtx;
 export let globalOpt: KAPLAYOpt;
@@ -334,10 +334,10 @@ const kaplay = <
     // if root is not defined (which falls back to <body>) we assume user is using kaboom on a clean page, and modify <body> to better fit a full screen canvas
     if (root === document.body) {
         document.body.style["width"] = "100%";
-        document.body.style["height"] = "100%";
+        document.body.style["height"] = "50%";
         document.body.style["margin"] = "0px";
         document.documentElement.style["width"] = "100%";
-        document.documentElement.style["height"] = "100%";
+        document.documentElement.style["height"] = "50%";
     }
 
     // create a <canvas> if user didn't provide one
@@ -356,7 +356,7 @@ const kaplay = <
     }
     else {
         canvas.width = canvas.parentElement!.offsetWidth;
-        canvas.height = canvas.parentElement!.offsetHeight;
+        canvas.height = (canvas.parentElement!.offsetHeight) / 2;
     }
 
     // canvas css styles
@@ -373,7 +373,7 @@ const kaplay = <
     }
     else {
         styles.push("width: 100%");
-        styles.push("height: 100%");
+        styles.push("height: 50%");
     }
 
     if (gopt.crisp) {
@@ -679,8 +679,8 @@ const kaplay = <
     const query = game.root.query.bind(game.root);
     const tween = game.root.tween.bind(game.root);
 
-    kaSprite = loadSprite(null, kaSpriteSrc);
-    boomSprite = loadSprite(null, boomSpriteSrc);
+    // kaSprite = loadSprite(null, kaSpriteSrc);
+    // boomSprite = loadSprite(null, boomSpriteSrc);
 
     function fixedUpdateFrame() {
         // update every obj
